### ISSUE TITLE
No Indicator on Initial View

### ISSUE THREAD

**Issue by jminutaglio** [DDraiman1990/DroidTimeSelection#13]
First: **ty for the 2.0 version - awesome.**

On initial view, there is no indicator:
<img width="344" alt="Screen Shot 2021-03-28 at 11 59 02 PM" src="https://user-images.githubusercontent.com/49879291/112785296-aa468180-9021-11eb-9d99-50670563e683.png">
```

        var style = HybridStyle()
        style.picker.titleColor = .white
        style.clock.indicatorColor = .purple
        style.modeButtonTint = .purple
        style.submitButtonColor = .green
        style.cancelButtonColor = .red
        
        
        let vc = DroidFactory.Hybrid.viewController(timeFormat: .twelve, style: style)
        
        vc.selector.onCancelTapped = {
            vc.dismiss(animated: true, completion: nil)
        }
                
        vc.selector.onOkTapped = {    
            let value = vc.selector.time //the time selected
            print("selected time:", value)
                        
            vc.dismiss(animated: true, completion: nil)
        }
                
        vc.selector.onSelectionChanged = { [weak self] value in
        }
        
        present(vc, animated: true, completion: nil)


```

**Comment by DDraiman1990** [comment]
I apologize I rushed and missed this.
I will address this tonight and also add the ability to set an initial time to allow the capability to edit an existing time. Otherwise the selector will always start out reset.
Stay tuned

**Comment by DDraiman1990** [comment]
Fixed in version 2.0.1 which is now live. Thanks for bringing this up.
